README.md

```diff
@@ -16,9 +16,16 @@ Also called the Empower Plant UI/UX. This project was bootstrapped with [Create
 | sentry-logback | 5.5.1 |
 
 ## Setup
-1. Permit your IP address in CloudSQL.
+1. Permit your IP address in CloudSQL by posting a message in the `#se-team` Slack channel.
```

Comment:
Talked to Dustin & Simon about pointing out a Slack channel here.

Reply:
Let's make it:
1. Permit your IP address in CloudSQL
    - Go to https://console.cloud.google.com/sql/instances
    - You will see 1 instance, under Actions column click "..." -> Edit
    - Expand "Connections", under Authorized Networks click "ADD NETWORK"
    - Google "my IP address", add it.

Reply:
ooo I like this even better! Updating now...